doc/marlowe/tutorials/marlowe-data.rst

```diff
@@ -291,7 +329,7 @@ How does this affect the processing of a Marlowe contract? Each step is
 processed relative to a slot interval, and the current slot value needs
 to lie within that interval.
 
-The endpoints of the interval are accessible as the values
+The endpoints of the interval are accessible as used above, the values
```

Comment:
Probably needs another `as` or something like that, maybe it would be best to put `as used above` in parenthesis